_data/investing.yml

```diff
@@ -180,4 +180,4 @@ websites:
       url: https://www.wisebanyan.com/
       twitter: WiseBanyan
       img: wisebanyan.png
-      tfa: No
+      tfa: No
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.